libraries/standard/mqtt/test/unit/iot_tests_mqtt_platform.c

```diff
@@ -639,7 +639,6 @@ TEST_SETUP( MQTT_Unit_Platform )
 TEST_TEAR_DOWN( MQTT_Unit_Platform )
 {
     IotMqtt_Cleanup();
-    IotTestNetwork_Cleanup();
```

Comment:
Please see my other comment on Unity framework architecture

Reply:
It looks like this is being done correctly now, the network init/cleanup is only being done in the `MQTT_System_Platform` setup/teardown & unit tests are using the `MQTT_Unit_Platform` group.

Ideally unit tests should be in a different file, invoked using a different make target, etc. than functional/integration tests but that's beyond the scope of this review.